chain/blockchain.go

```diff
@@ -57,6 +57,7 @@ func (bc *Blockchain) AddBlock(block *block.Block, fastAdd bool) error {
 
 	err := bc.SaveBlock(block, fastAdd)
 	if err != nil {
+		log.Error("AddBlock error, ", err)
```

Comment:
Why `SaveBlock` error but log output "**AddBlock** error"?

Reply:
because this statement is in AddBlock function.